tests/sanity/ignore-2.9.txt

```diff
@@ -1,16 +0,0 @@
-plugins/modules/digital_ocean.py validate-modules:deprecation-mismatch
```

Comment:
The `deprecation-mismatch` lines in `ignore-2.9.txt` need to stay.

Reply:
(`invalid-documentation` and `missing-main-call` for the deprecated modules as well, probably.)